scripts/install-deps.sh

```diff
@@ -432,9 +432,13 @@ main()
         pop)
             install_deps_popos
             ;;
-        ubuntu|neon|debian)
+        ubuntu|neon)
             install_deps_ubuntu
             ;;
+        debian)
+            install_deps_ubuntu
+            fetch_and_unpack_fmtlib        
```

Comment:
Oha, interesting. And that's because Debian ships with a too told fmtlib, I assume?

Reply:
Indeed